web/app/webpack.config.js

```diff
@@ -64,10 +66,17 @@ module.exports = smp.wrap({
   },
   plugins: [
```

Comment:
I think the order of the plugins matters (https://stackoverflow.com/questions/41470771/webpack-does-the-order-of-plugins-matter); not that it matters for the plugins we currently have, but maybe the generation of the index_bundle and the moving of it should be in adjacent steps?

also, wdyt of adding comments in the `plugins` section so that it's more clear what each step does?

```suggestion
  plugins: [
    // new BundleAnalyzerPlugin(), // uncomment to analyze bundle size
    new CleanWebpackPlugin(['dist']),
    new LodashModuleReplacementPlugin({
      // 'chain': true,
      'collections': true,
      'paths': true
    }),
    // compile the bundle with hashed filename into index_bundle.js.out
    new HtmlWebpackPlugin({
      inject: false,
      filename: "index_bundle.js.out",
      template: 'index_bundle.js.lodash.tmpl',
    }),
    // move /dist/index_bundle.js.out to /dist/index_bundle.js
    new WebpackMvPlugin()
  ]
```